components/footer.js

```diff
@@ -90,10 +86,10 @@ const Footer = ({ intl }) => (
         </FooterBox>
       </Flex>
       <Flex flexWrap='wrap'>
-        <FooterBox width={[1, 2/6]}>
+        <FooterBox>
           <small>
             <Box mb={1}>{intl.formatMessage({ id: 'Footer.Text.Copyright' })}</Box>
-            <Box>{intl.formatMessage({ id: 'Footer.Text.CCommons' })}</Box>
+            <FooterLink href='https://github.com/ooni/license' label={intl.formatMessage({ id: 'Footer.Text.CCommons'})} />
```

Comment:
We should maybe host somewhere a non github version of all our licenses, but I guess this is good for now.